lightkurve/data/tpf-ext1-header.txt

```diff
@@ -0,0 +1,282 @@
+NAXIS1  =                  220 / length of first array dimension                
+NAXIS2  =                 3470 / length of second array dimension               
+PCOUNT  =                    0 / group parameter count (not used)               
+GCOUNT  =                    1 / group count (not used)                         
+TFIELDS =                   13 / number of table fields                         
+TTYPE1  = 'TIME    '           / column title: data time stamps                 
+TFORM1  = 'D       '           / column format: 64-bit floating point           
+TUNIT1  = 'BJD - 2454833'      / column units: barycenter corrected JD          
+TDISP1  = 'D14.7   '           / column display format                          
+TTYPE2  = 'TIMECORR'           / column title: barycenter - timeslice correction
+TFORM2  = 'E       '           / column format: 32-bit floating point           
+TUNIT2  = 'd       '           / column units: day                              
+TDISP2  = 'E14.7   '           / column display format                          
+TTYPE3  = 'CADENCENO'          / column title: unique cadence number            
+TFORM3  = 'J       '           / column format: signed 32-bit integer           
+TDISP3  = 'I10     '           / column display format                          
+TTYPE4  = 'RAW_CNTS'           / column title: raw pixel counts                 
+TFORM4  = '8J      '           / column format: image of signed 32-bit integers 
+TUNIT4  = 'count   '           / column units: count                            
+TDISP4  = 'I8      '           / column display format                          
+TDIM4   = '(1,8)   '           / column dimensions: pixel aperture array        
+TNULL4  =                   -1 / column null value indicator                    
+WCSN4P  = 'PHYSICAL'           / table column WCS name                          
+WCAX4P  =                    2 / table column physical WCS dimensions           
+1CTY4P  = 'RAWX    '           / table column physical WCS axis 1 type, CCD col 
+2CTY4P  = 'RAWY    '           / table column physical WCS axis 2 type, CCD row 
+1CUN4P  = 'PIXEL   '           / table column physical WCS axis 1 unit          
+2CUN4P  = 'PIXEL   '           / table column physical WCS axis 2 unit          
+1CRV4P  =                  581 / table column physical WCS ax 1 ref value       
+2CRV4P  =                   90 / table column physical WCS ax 2 ref value       
+1CDL4P  =                  1.0 / table column physical WCS a1 step              
+2CDL4P  =                  1.0 / table column physical WCS a2 step              
+1CRP4P  =                    1 / table column physical WCS a1 reference         
+2CRP4P  =                    1 / table column physical WCS a2 reference         
+WCAX4   =                    2 / number of WCS axes                             
+1CTYP4  = 'RA---TAN'           / right ascension coordinate type                
```

Comment:
By using a template header with a WCS in it, are you not artificially adding the wrong WCS to your new data product? Would it be better to have these values as NaNs or 0's in the template just incase they don't get overwritten?

Reply:
I think you're right. When I originally made the template file, I had intended to get rid of the WCS keywords and only write them in later. Maybe we should do that?